cohere/responses/classify.py

```diff
@@ -8,19 +8,34 @@
 
 class Classification(CohereObject):
     def __init__(
-        self, input: str, prediction: str, confidence: float, labels: Dict[str, LabelPrediction], *args, **kwargs
+        self,
+        input: str,
+        predictions: Optional[List[str]],
+        confidences: Optional[List[float]],
+        prediction: Optional[str],
+        confidence: Optional[float],
+        labels: Dict[str, LabelPrediction],
+        *args,
+        **kwargs,
     ) -> None:
         super().__init__(*args, **kwargs)
         self.input = input
-        self.prediction = prediction
-        self.confidence = confidence
+        self.prediction = prediction  # to be deprecated
```

Comment:
how do you expect to deprecate this field in the future?

Reply:
@lfayoux You're probably better positioned to answer this question than me. I'm not sure what's the standard procedure for field deprecation.

Reply:
This will be deprecated when we do an api version change. Not sure if we still want to support this field in the future

Reply:
Took that into account

Reply:
logging warnings for now is ok. when we deprecate we will need to bump the major SDK version